databases/week1/database.sql

```diff
@@ -0,0 +1,70 @@
+--1.Find out how many tasks are in the task table
+
+SELECT COUNT(*)
+FROM HYF.task;
+
+--.Find out how many tasks in the task table do not have a valid due date
+
+SELECT COUNT(*)
+FROM task
+WHERE due_date IS NULL;
+
+--3.Find all the tasks that are marked as done
+
+SELECT *
+FROM task t
+    INNER JOIN status s
+    ON t.status_id=s.id
+WHERE s.name="Done";
```

Comment:
I think this is fine, but I am curios why you need an alias here? Why not:

    SELECT *
    FROM task
      INNER JOIN status
      ON task.status_id = status.id
    WHERE status.name = "Done";